tests/installation/addon_products_sle.pm

```diff
@@ -110,7 +110,10 @@ sub handle_addon {
     # SES6 on SLE15 in development has untrusted key warning
     addon_license($addon) unless is_sle('15+') && $addon !~ /^ses$|^rt$/;
     # might involve some network lookup of products, licenses, etc.
-    assert_screen 'addon-products', 90;
+    assert_screen ['addon-products', 'import-untrusted-gpg-key'], 90;
+    if (match_has_tag('import-untrusted-gpg-key')) {
+        send_key 'alt-t';
+    }
```

Comment:
You might want to `assert_screen 'addon-products'` after `alt-t` to make sure you don't `send-key tab` too soon

Reply:
Don't blindly accept that - this is a bug if not in development!

Reply:
It's already blindly accepted for [ADDONURL](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/addon_products_sle.pm#L196)
@kravciak had idea to add workaround tag on that needle

Reply:
That's a valid point. @dzedro do you have scenarios where you can clearly say that it's not development version? E.g. when we boot into GM image? For Functional job group we don't deal with non-dev versions, I believe that's the reason why we have this behavior for ADDONURL. Will you follow up on that in this PR?

Reply:
If we don't care in functional if ADDONURL does have untrusted key then why we start to care on devel if it is using untrusted key, when iso is the same content as repo ? I miss the logic
Personally I would like to avoid this by using ADDONURL, but undortunatelly it's more  complicated ><

Reply:
My point was about e.g. migration scenarios, where it could be a bug. But at least it's also not your scenario, so let's leave this for now and let responsible team come up with an improvement. Cheers ;)

Reply:
I could change it to `(match_has_tag('import-untrusted-gpg-key') && get_var('DEVELOPMENT'))` or something to avoid blindly accepting non-devel products, but I will at least add the workaround tag on the needle